apps/docs/src/services/auth.ts

```
@@ -0,0 +1,209 @@
+/**
+ * Authentication and Database Services
+ *
+ * Azure-based implementation for user authentication and data storage.
+ * Uses Azure Entra ID (MSAL) for auth and Azure Functions/Cosmos DB for data.
+ */
+
+import {
+  getCloudServices,
```

ESLint

[error] 'getCloudServices' is defined but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)